backend/Recodo.API/Controllers/UserController.cs

```diff
@@ -1,22 +1,39 @@
-﻿using Microsoft.AspNetCore.Authorization;
-using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Recodo.API.Extensions;
 using Recodo.BLL.Services;
+using Recodo.Common.Dtos.User;
 using System.Threading.Tasks;
 
 namespace Recodo.API.Controllers
 {
-    [Route("api/users")]
     [ApiController]
+    [Route("api/User")]
     public class UserController : ControllerBase
     {
-        private UserService _userService;
-        public UserController(UserService userService)
+        private readonly AuthService _authService;
+        private readonly UserService _userService;
+
+        public UserController(AuthService authService, UserService userService)
         {
+            _authService = authService;
             _userService = userService;
         }
 
+        [HttpPost("ResetPassword")]
+        public async Task<IActionResult> ResetPassword(UpdateUserDTO user)
+        {
+            await _userService.ResetPassword(user.Email);
+            return NoContent();
+        }
+
+        [HttpPost("ResetPasswordDone")]
```

Comment:
Rename to process (for now it's a finished action). The same for methods. They should show what they do, not in what they will result

Reply:
Renamed